syntax_and_semantics/annotations.md

```diff
@@ -78,6 +78,39 @@ end
 annotation_value # => 2
 ```
 
+The `named_args` method can be used to read all key/value pairs on an annotation as a `NamedTupleLiteral`.
```

Comment:
Maybe add that these don't need to be declared in the annotation type and that different instantiations (is that the correct term?) of an annotation can have different arguments.
Ditto for `args`.